src/index.html

```diff
@@ -7,8 +7,57 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
+
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div class="icon">
+        <a href="#home" class="icon_link">
+          <img
+            src="images/logo.png"
+            alt="Moyo_icon"
+            class="icon"
+          >
+        </a>
+      </div>
+      <nav class="nav header_nav">
+        <ul class="nav_list">
+          <li class="nav_item">
+            <a href="#" class="nav_link is-active">Apple</a>
+          </li>
+          <li class="nav_item">
+            <a href="#" class="nav_link">Samsung</a>
+          </li>
+          <li class="nav_item">
+            <a href="#" class="nav_link">Smartphones</a>
+          </li>
+          <li class="nav_item">
+            <a
+              href="#"
+              class="nav_link"
+              data-qa="hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+          <li class="nav_item">
+            <a href="#" class="nav_link">Gadgets</a>
+          </li>
+          <li class="nav_item">
+            <a href="#" class="nav_link">Tablets</a>
+          </li>
+          <li class="nav_item">
+            <a href="#" class="nav_link">Photo</a>
+          </li>
+          <li class="nav_item">
+            <a href="#" class="nav_link">Video</a>
+          </li>
+        </ul>
+
```

Comment:
Add spaces between elements inside the parent tag, it will improve the readability of your code.


```suggestion
      <nav class="nav header_nav">
        <ul class="nav_list">
          <li class="nav_item">
            <a href="#" class="nav_link is-active">Apple</a>
          </li>
          
          <li class="nav_item">
            <a href="#" class="nav_link">Samsung</a>
          </li>
          
          <li class="nav_item">
            <a href="#" class="nav_link">Smartphones</a>
          </li>
          
          <li class="nav_item">
            <a
              href="#"
              class="nav_link"
              data-qa="hover"
            >
              Laptops & Computers
            </a>
          </li>
          
          <li class="nav_item">
            <a href="#" class="nav_link">Gadgets</a>
          </li>
          
          <li class="nav_item">
            <a href="#" class="nav_link">Tablets</a>
          </li>
          
          <li class="nav_item">
            <a href="#" class="nav_link">Photo</a>
          </li>
          
          <li class="nav_item">
            <a href="#" class="nav_link">Video</a>
          </li>
        </ul>
```